plugins/package-managers/go/src/test/kotlin/GoModTest.kt

```diff
@@ -69,4 +71,19 @@ class GoModTest : WordSpec({
             )
         }
     }
+    "escapeVersion" should {
```

Comment:
Please add an empty line before this one.

Reply:
Done